initials.c

```diff
@@ -0,0 +1,10 @@
+#include<stdio.h>
```

Comment:
Great code! The only thing I'd add is your name and what this program does at the top (in comments). Other than that, you followed all the guidelines and your code runs!